content/terms/archives/cloud/2022-09-14.md

```diff
@@ -5,65 +5,124 @@ title: Terms of Service for Sourcegraph Cloud (Archived)
 
 Last modified: September 14, 2022
 
-See the [changes](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/about%24+file:%5Econtent/terms/cloud%5C.md%7C%5Edocs/terms-dotcom+type:diff&patternType=standard) since the [previous version](/terms/archives/cloud/2022-02-09/) or visit our [archives](https://github.com/sourcegraph/about/tree/main/content/terms/archives).
+See the [changes](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/about%24+file:%5Econtent/terms/cloud%5C.md%7C%5Edocs/terms-dotcom+type:diff&patternType=standard) since the [previous version](/terms/archives/cloud/2022-09-14/) or visit our [archives](https://github.com/sourcegraph/about/tree/main/content/terms/archives).
```

Comment:
The previous version shouldn't link to the current version it's on, no? @tammy-zhu

Reply:
@st0nebraker you're right. i've updated the previous version link to the 2022-02-09 version. thanks for catching!